README.md

```diff
@@ -563,8 +573,24 @@ ort {
 }
 ```
 
-Currently [Nexus IQ Server](https://help.sonatype.com/iqserver) (`-a NexusIQ`) is the only supported security data
-provider.
+To enable this provider, pass `-a NexusIQ` on the command line.
+
+## VulnerableCode
+
+This provider obtains information about security vulnerabilities from a
+[VulnerableCode](https://github.com/nexB/vulnerablecode) instance. The configuration is limited to the server URL, as
+authentication is not required:
+
+```hocon
+ort {
+  advisor {
+    VulnerableCode {
+      serverUrl = "http://localhost:8000"
+    }
+  }
+}
+```
+To enable this provider, pass `-a VulnerableCode` on the command line.
```

Comment:
Nit: Please add a blank line before this one, like in the NexusIQ example above.